install_config/install/advanced_install.adoc

```diff
@@ -248,6 +248,9 @@ master may require access to, or the installation will fail. Defaults to
 |This variable overrides the default subdomain to use for exposed
 xref:../../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[routes].
 
+|`openshift_master_image_policy_config`
+|Sets `imagePolicyConfig` in the master configuration. See xref:../../install_config/master_node_configuration.adoc#master-config-image-config[Image Configuration] for details.
+
```

Comment:
This is only in here cuz the original PR was adding it along w/ the web console settings. Since it doesn't really fit w/ the others I moved it up here.